src/parser.rs

```diff
@@ -188,6 +188,11 @@ impl<'i: 't, 't> Parser<'i, 't> {
         }
     }
 
+    /// Return the current line that is being parsed.
```

Comment:
Gecko's error messages include a `sourceLine` property which is expected to be the current line being parsed.